src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -443,7 +452,7 @@ public static void DropDatabase(SqlConnection sqlConnection, string dbName)
             }
         }
 
-        public static bool IsLocalDBInstalled() => !string.IsNullOrEmpty(LocalDbAppName?.Trim());
+        public static bool IsLocalDBInstalled() => !string.IsNullOrEmpty(LocalDbAppName?.Trim()) && !string.IsNullOrEmpty(LocalDbSharedInstanceName?.Trim());
```

Comment:
Could you split this check into 2?
- IsLocalDBInstalled
- IsSharedLocalDBInstalled

So we could still run local db tests without shared local db setup available.

Reply:
done.